examples/platform/qpg6100/comps/gpFreeRTOS/config/hooks.c

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
Is this file copied in the repository without modifications?  Why?

Should we be pointing the build at a Qorvo SDK instead of checking these files into the repo?

Reply:
At this stage we do not have a publicly available location for the SDK library and supporting files.
As an intermediate step we'd like to add them here to move them out when we have a proper location available.
We also based ourselves on the approach taken in OpenThread where the needed code does reside in the repo itself in a /third_party/Qorvo folder
Likewise we can move the files into a /third_party/qorvo_sdk  if that is a better approach ?